displayTable.py

```diff
@@ -5,9 +5,10 @@
 	@param wiki - Language code for the wiki (eg: 'en', 'es', 'it')
 	@param content - Python 2D array (nested list) containing table entries
 	@param desc - Key to the description message of the report
+	@param numbers - If table have to contain numbers of rows
```

Comment:
Not yet used.
My main idea was to create an option to allow to add lines number in tables. I've reverted it and for the time being preferred hard code it for each report that need it.